contract/test/authRole_test.js

```diff
@@ -64,93 +56,89 @@ contract("authRole", function (accounts) {
   it("Make a user as MAINTAINER", async () => {
     const authRoleInstance = await authRole.deployed();
     const authRoleAddNewTableContract = await authRoleInstance.addNewTableContract.sendTransaction(
-      skOwnerTest,
       tableNameTest
     );
 
     const authGrantPermission = await authRoleInstance.grantPermission.call(
-      skOwnerTest,
-      skMaintainerTest,
+      accounts[1],
       tableNameTest,
       maintainerRole
     );
-    assert.equal(authGrantPermission, true, "Assign skTest as MAINTAINER");
+    assert.equal(
+      authGrantPermission,
+      true,
+      "Assign `accounts[1]` as a MAINTAINER"
```

Comment:
Use "grant"

Reply:
`accounts[1]` might be inconvenient. How about introducing mnemonic terms?

Reply:
How about using `a given address` to replace `accounts[1]`?

Reply:
It sounds good. You only have to mention the exact address in the very beginning.

Reply:
`accounts[]` is a list of ethereum addresses returned by the test framework. The addresses are generated every time we run the test, so actually, we have no idea what the address exact is

Reply:
You can still prompt the value of `accounts` at first glance. Then, we don't have to embed its value later.

Reply:
OK